dnf/cli/output.py

```diff
@@ -1963,7 +2037,8 @@ def historyPackageListCmd(self, extcmds):
         # REALLY Needs to use columns!
         print(fmt % (fill_exact_width(_("ID"), 6, 6),
                      fill_exact_width(_("Action(s)"), 14, 14),
-                     fill_exact_width(P_("Package", "Packages", 1), 53, 53)))
+        # This is also a hack to resolve RhBug 1302935 correctly.
+                     fill_exact_width(C_("long", "Package"), 53, 53)))
```

Comment:
again plural forms are lost

Reply:
The same explanation here. The original cause is https://bugzilla.redhat.com/show_bug.cgi?id=1302935. It turned out that gettext does not handle correctly the cases where the same word (```"Package"```) appears both standalone and in the plural context (```"Package"```/```"Packages"```). As a workaround all occurrences of ```_("Package")``` have been replaced with ```P_("Package", "Packages", 1)```. However, this solution is still not perfect because translators want to translate ```"Package"``` standalone (e.g., in a header) with uppercase while ```"%d Package(s)"``` with lowercase (see [comment 15](https://bugzilla.redhat.com/show_bug.cgi?id=1302935#c15)). The correct solution could be to use contexts: ```"Package"``` in a header vs. ```"Package"```/```"Packages"``` with a number. But since we are already using contexts for short vs. long versions of the word ```"Package"``` let's use the same context to resolve the old issue. Again, there is no additional code for 1302935, just a comment that the same code also resolves the other issue.

Reply:
Thanks for explanation, in that case I'm merging the PR.